### ISSUE TITLE
Instance Creation Failed

### ISSUE THREAD

**Issue by viiuan** [Tre5et/treelauncher#10]
![image](https://github.com/user-attachments/assets/ee531332-ce93-45c6-bcb6-babbc573815f)
![image](https://github.com/user-attachments/assets/ba2feef2-94b4-4d78-896d-a42af6f659c5)
Happens when creating instances (tested so far): 1.0, 1.1, 1.2.1, 1.3.1, 1.4.2, 1.5.1.
Component settings do not matter.

Going back to the instances page gets you this error.
![image](https://github.com/user-attachments/assets/27d38852-44cf-4a6b-8a3e-17a1e63bdfa8)
[2024-08-01_2.log.gz](https://github.com/user-attachments/files/16459904/2024-08-01_2.log.gz)

After getting this instance error, this is what happens when you open the launcher and log in.
![image](https://github.com/user-attachments/assets/8bd53f19-5e5d-4993-bfc8-0cceb12dbdb6)

Originally occurred on 2.3.0 (no log), then 2.5.0:
[2024-08-01_5.log.gz](https://github.com/user-attachments/files/16459786/2024-08-01_5.log.gz)

DId not occur when creating instances: Fabric 1.20.1, Fabric 1.21.

It also seems that the launcher gets stuck on the initial setup at 100%, I have to restart it to proceed.

**Comment by Tre5et** [comment]
Hey, thanks for the detailed report! Also, sorry that you are experiencing this many errors.

I haven't verified the launcher to work with such old versions. It may be that the installation process has changed over the years in a way I didn't consider. I will look into this, but depending on the complexity, it may take a little time until I have worked out complete support for these versions.

As for the getting stuck on 100%, I've opened a second issue, but I expect this to be a rather quick fix.
